src/drawmembrane/draw_membrane2a.c

```diff
@@ -541,6 +580,12 @@ int main(int argc, char *argv[])
   R_m1 = 0;      /* upper exclusion radius */
   R_m0 = 0;      /* lower exclusion radius */
 
+  /*11-2021*/
+  /* Warning: this is the hard coded offset. Normally you should get them from core.py*/
+  /* But the way wrote in core.py made it hard to do so. */
+  dx_R = 20;
+  dy_R = -5;
+
   opterr = 0;
   while ((c = getopt(argc, argv, "hZz:d:s:c:m:p:i:a:V:I:r:R:X:Y:")) != -1) {
```

Comment:
add options for reading dx_R and dy_R, maybe `x:y:D` for flags `-x dx_R -y dy_R -D` (where `-D` activates the *dimer* code)?